### DISCUSSION TITLE
Bookmarks synced with local browser bookmarks

### DISCUSSION THREAD

**Discussion by ozerkerr** [dailydotdev/daily#1511] [Feature Requests]
Hello DailyDev,

I would like to request a new feature for our platform: the ability to sync bookmarks between the platform and the browser’s local bookmarks. This would greatly improve usability by allowing users to seamlessly access and manage their saved bookmarks, both on the platform and directly through their browser's native bookmarking system.

Proposed Features:

- Sync bookmarks automatically between the platform and browser’s local bookmarks (Chrome, Firefox, Edge, etc.).
- Provide an option for users to toggle this sync on or off.
- Allow for real-time updating when bookmarks are added, removed, or modified on either side.
- Ensure compatibility across all major browsers.
- Option to import/export bookmarks manually for additional control.

This feature would enhance the user experience by making bookmarking more efficient and integrated. Looking forward to your thoughts!

Best regards,
Ozer Kelgem

**Comment by idoshamun** [comment]
Hi @ozerkerr,

Thanks for the suggestion. We will consider it
